<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gp_gestaovendas</class>
 <widget class="QDialog" name="gp_gestaovendas">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>623</width>
    <height>408</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDateEdit" name="de_dataini">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateEdit" name="de_datafim">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>30</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_imprimirgvenda">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>360</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Imprimir</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tw_listavendas">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>581</width>
     <height>131</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="tw_listaprodvendas">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>581</width>
     <height>151</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Data Inicial</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>10</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Data Final</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>30</y>
     <width>231</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="btn_filtrargvenda">
      <property name="text">
       <string>Filtrar</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Mostrar todas vendas</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
